packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1668,6 +1668,22 @@ class Positioned extends ParentDataWidget<Stack> {
        bottom = null,
        super(key: key, child: child);
 
+  /// Creates a Positioned object with the values from the given [RelativeRect].
+  ///
+  /// This sets the [left], [top], [right], and [bottom] properties from the
+  /// given [RelativeRect]. The [height] and [width] properties are set to null.
+  Positioned.fromRelativeRect({
+    Key key,
+    Widget child,
```

Comment:
Isn't the child @required (likewise below)? Also: if you're willing to be obsessive, the Positioned constructor lists the required child last.